copyspecial.py

```diff
@@ -13,29 +13,65 @@
 import commands
```

Comment:
The commands module was depreciated in Python 2.6. Use the subprocess module instead. It exists in Python 2.6 to current versions of Python 3

Reply:
The commands and sys modules are imported but not used.